src/shared/services/map-location/map-location.test.ts

```diff
@@ -21,56 +21,22 @@ const testAddress = {
   woonplaats: 'Amsterdam',
 }
 
-const testLocation = { lng: 4, lat: 52 }
+const coordinates = { lng: 4, lat: 52 }
 
 const testFeature = {
   type: 'Point',
-  coordinates: [4, 52],
+  coordinates: [coordinates.lat, coordinates.lng] as LatLngTuple,
 }
 
 describe('locationToFeature', () => {
   it('should convert', () => {
```

Comment:
maybe we should rename these methods to coordinatesToFeature, and featureToCoordinates?

Reply:
Absolutely. Good catch. Is it okay if I tackle that in an upcoming PR?